client/app/account/component/AccountInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { MdAccountCircle } from 'react-icons/md'
+import { colorMap } from '@/_consts/iconColor'
+import { UserType } from '@/_types'
+
+type AccountInfoProps = {
+  user: UserType
+}
+
+const AccountInfo = ({ user }: AccountInfoProps) => {
+  return (
+    <div className='flex flex-col gap-6 font-semibold'>
```

Comment:
Hmm, I feel all texts are too thick. I think the title is font-semibold, and the text is font-normal is better. What do you think? 

<img width="409" alt="Screenshot 2024-03-14 at 4 53 13 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/0a205692-84a3-4d45-86d0-61f78f88532c">

Reply:
Fixed it
7a54c48eb4bacceb0b71c5abeac73c91f6ccd071